src/buildCommon.js

```diff
@@ -26,6 +26,7 @@ var greekCapitals = [
 var dotlessLetters = [
```

Comment:
I think I'll go for `mainitLetters`, since in the context of the `mathit` function, the emphasis is not on italic but on main instead of math.